tools/fuchsia/devshell/serve.sh

```diff
@@ -175,9 +245,16 @@ while true; do
     fi
 
     config_url="http://${addr}:${port}/config.json"
-    run_ssh_command pkgctl add \
-      -n "engine" \
-      "${config_url}"
+
+    if [[ ${component_framework_version} == 2 ]]; then
+      run_ssh_command pkgctl add \
```

Comment:
I messed up this command, it should be "pkgctl repo add". I was running serve.sh incorrectly to test so I didn't notice.

Reply:
done